akka-testkit/src/main/scala/akka/testkit/ExplicitlyTriggeredScheduler.scala

```diff
@@ -33,10 +34,10 @@ import com.typesafe.config.Config
 class ExplicitlyTriggeredScheduler(@unused config: Config, log: LoggingAdapter, @unused tf: ThreadFactory)
     extends Scheduler {
 
-  private case class Item(time: Long, interval: Option[FiniteDuration], runnable: Runnable)
+  private case class Item(id: UUID, interval: Option[FiniteDuration], runnable: Runnable)
```

Comment:
Alternatively we could make this a regular non-case class, that way we don't need the `id` field. That seems a bit simpler, what do you think?

Reply:
Sure, the regular class version looks better. I changed it as well.